### PR TITLE
gce provider fixes

### PR DESCRIPTION
**Pull request by cooktheryan**
Part of the fix required for #248

### CODE REVIEW

**Comment by cooktheryan** [comment]
@detiber is this the correct path for the cloud config information you referenced in #242

File: reference-architecture/gce-ansible/playbooks/openshift-setup.yaml [hunk 1]
```diff
@@ -6,23 +6,18 @@
...
-      - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/gce/gce.conf"
     openshift_node_kubelet_args:
-      cloud-provider:
-      - "gce"
       cloud-config:
-      - "/etc/origin/gce.ini"
+      - "/etc/gce/gce.conf"
```
Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/openshift-setup.yaml]: you shouldn't need to set osm_controller_args, osm_api_server_args, or openshift_node_kubelet_args. you'll probably need to munge the created cloud config file put down, but we should really add the support needed directly to openshift-ansible.

File: reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
-- name: ensure the /etc/orgin folder exists
-  file: name=/etc/origin state=directory
+- name: ensure the /etc/origin/cloudprovider folder exists
+  file: name=/etc/origin/cloudprovider state=directory recurse=yes
 
 - name: configure gce cloud config options
-  ini_file: dest=/etc/origin/gce.ini section=global option={{ item.key }} value={{ item.value }} state=present create=yes
+  ini_file: dest=/etc/origin/cloudprovider/gce.conf section=global option={{ item.key }} value={{ item.value }} state=present create=yes
```
Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]: I think we will clobber the multizone value in this file with openshift-ansible, which isn't an issue unless you are deploying to only a subset of the zones in the region.

Comment (cooktheryan) [inline reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]: I believe we are single AZ right now

Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]: @cooktheryan that would be an issue then, you would need multizone=False in that case

Comment (detiber) [inline reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]: see here: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_cloud_provider/tasks/gce.yml

Comment (cooktheryan) [inline reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]: I think we have it just a little bit down further in the file

[https://github.com/cooktheryan/openshift-ansible-contrib/blob/49d1dcbffc81fedfcf17dc25bf389b5470645e19/reference-architecture/gce-ansible/playbooks/roles/gce-cloudconfig/tasks/main.yaml]

**Comment by cooktheryan** [comment]
Validation
master-config.yaml
```
kubernetesMasterConfig:
  apiServerArguments: 
    cloud-config:
    - /etc/origin/cloudprovider/gce.conf
    cloud-provider:
    - gce
  controllerArguments: 
    cloud-config:
    - /etc/origin/cloudprovider/gce.conf
    cloud-provider:
    - gce
```

node-config.yaml
```
kubeletArguments: 
  cloud-config:
  - /etc/origin/cloudprovider/gce.conf
  cloud-provider:
  - gce
  node-labels:
  - role=master
```

```
PLAY RECAP *********************************************************************
localhost                  : ok=23   changed=8    unreachable=0    failed=0   
ocp-infra-38hx             : ok=177  changed=55   unreachable=0    failed=0   
ocp-infra-b328             : ok=177  changed=55   unreachable=0    failed=0   
ocp-master-725w            : ok=562  changed=185  unreachable=0    failed=0   
ocp-master-h817            : ok=362  changed=118  unreachable=0    failed=0   
ocp-master-j3lf            : ok=362  changed=118  unreachable=0    failed=0   
ocp-node-10fz              : ok=172  changed=58   unreachable=0    failed=0   
ocp-node-h6v1              : ok=172  changed=58   unreachable=0    failed=0  
```

**Pull request merged**